PSReadLine/Keys.cs

```diff
@@ -304,12 +304,12 @@ static ConsoleKeyInfo CtrlAlt(char c)
         public static ConsoleKeyInfo CtrlE               = Ctrl('\x05');
         public static ConsoleKeyInfo CtrlF               = Ctrl('\x06');
         public static ConsoleKeyInfo CtrlG               = Ctrl('\a');
-        public static ConsoleKeyInfo CtrlH               = Ctrl('\b');
-        public static ConsoleKeyInfo CtrlI               = Ctrl('\t');
-        public static ConsoleKeyInfo CtrlJ               = Ctrl('\n');
+        public static ConsoleKeyInfo CtrlH               = Ctrl('\b'); // !Linux, generate (keychar: '\b', key: Backspace, mods: 0), same as CtrlBackspace
+        public static ConsoleKeyInfo CtrlI               = Ctrl('\t'); // !Linux, generate (keychar: '\t', key: Tab,       mods: 0)
+        public static ConsoleKeyInfo CtrlJ               = Ctrl('\n'); // !Linux, generate (keychar: '\n', key: Enter,     mods: 0)
```

Comment:
This feels like a CLR bug - `showkey -a` returns 10 for Ctrl+j and 13 for Ctrl+m or Enter.

Reply:
True, and good to know the `showkey` command. I will update the comment to point out it's a CLR bug.

Reply:
The comment to `CtrlM` is updated to: `// !Linux, same as CtrlJ but 'showkey -a' shows they are different, CLR bug`